operator_claims.go

```diff
@@ -19,24 +19,31 @@ import (
 	"errors"
 	"fmt"
 	"net/url"
+	"strings"
 
 	"github.com/nats-io/nkeys"
 )
 
 // Operator specific claims
 type Operator struct {
-	Identities       []Identity `json:"identity,omitempty"`
-	SigningKeys      StringList `json:"signing_keys,omitempty"`
```

Comment:
I prefer not typing things that are basic Go, should just be []string.

Reply:
we'll have to add the uniquing code, empty string suppression, everywhere or at least on generate. Also all the clients will have to handle the removal of an element from the slice etc.

Reply:
ok